### PR TITLE
export hack/tools/bin folder to PATH

### PR DESCRIPTION
**Pull request by wongni**
*Issue #, if available:*

*Description of changes:*

The e2e test run fails with the following error if `kustomize` is not in the system PATH. 

```
  [91mFailed to generate the manifest for "infrastructure-cloudstack" / "v1.0.0"
  Unexpected error:
      <*errors.withStack | 0xc000110618>: {
          error: <*errors.withMessage | 0xc0000a0220>{
              cause: <*errors.withStack | 0xc0001104f8>{
                  error: <*exec.Error | 0xc0000a01c0>{
                      Name: "kustomize",
                      Err: <*errors.errorString | 0xc000252210>{
                          s: "executable file not found in $PATH",
                      },
                  },
                  stack: [0x1992867, 0x1a8bc9a, 0x1a8ae68, 0x1db16cb, 0x1db284b, 0x4d1a85, 0x4d1005, 0x7352fa, 0x73323a, 0x732c05, 0x7344fe, 0x734132, 0x7588c7, 0x758473, 0x75a732, 0x7668e5, 0x7666fe, 0x1db12ca, 0x515262, 0x46b181],
              },
              msg: "failed to execute kustomize: ",
          },
          stack: [0x1a8bd31, 0x1a8ae68, 0x1db16cb, 0x1db284b, 0x4d1a85, 0x4d1005, 0x7352fa, 0x73323a, 0x732c05, 0x7344fe, 0x734132, 0x7588c7, 0x758473, 0x75a732, 0x7668e5, 0x7666fe, 0x1db12ca, 0x515262, 0x46b181],
      }
      failed to execute kustomize: : exec: "kustomize": executable file not found in $PATH
  occurred[0m

```

*Testing performed:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by maxdrib — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -269,6 +269,7 @@ e2e-essentials: $(GINKGO_V1) e2e-cluster-templates kind-cluster ## Fulfill essen
 
 JOB ?= .*
 run-e2e: e2e-essentials ## Run e2e testing. JOB is an optional REGEXP to select certainn test cases to run. e.g. JOB=PR-Blocking, JOB=Conformance
+	export PATH=$(REPO_ROOT)/$(TOOLS_BIN_DIR):$(PATH) && \
 	kubectl apply -f cloud-config.yaml && \
```
Comment (maxdrib) [inline Makefile:273]: Would this work? It's defined at the top of the makefie
```suggestion
	$KUBECTL apply -f cloud-config.yaml && \
```

Comment (wongni) [reply Makefile]: fixed

**Review by maxdrib — APPROVED** [review]

**Pull request merged**